app/views/users/_user.html.erb

```diff
@@ -0,0 +1,11 @@
+<div class="user">
+	
+		<%= image_tag(user.photo, alt: "user photo", class: "user_photo") %>
+	
+		<div class="user_info">
+			<h2>
+			<%= link_to user.name, user_path(user.id) %>
+			</h2>		
+			<p>Number of posts: <%= user.posts_counter %></p>
+		</div>
+	</div>
```

Comment:
- [ ] **[OPTIONAL]** For better code readability I would recommend you properly indent your code.